airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_incremental.py

```diff
@@ -174,6 +178,9 @@ async def test_two_sequential_reads(
                         "stream": {"stream_descriptor": stream_descriptor, "stream_state": stream_state},
                     }
                 )
+        elif is_global_state(states_1[-1]):
```

Comment:
Nit: Might as well change the `is_per_stream = ` line above and make the `if` check consistent with how this line does it.

Question for the TODO comment: so we actually want different handling for global state that @xiaohansong adds to this block, or is it hooked in dynamically? 

My understanding is otherwise the behavior won't change if we merge this today.

Reply:
> Question for the TODO comment: so we actually want different handling for global state that @xiaohansong adds to this block, or is it hooked in dynamically?

We'll actually have different handling here.

> My understanding is otherwise the behavior won't change if we merge this today.

That's correct.

Reply:
Acked. thanks for making the change!